modules/dashboard/modules/widgets/container/network/base.tf

```diff
@@ -18,10 +19,8 @@ module "base" {
   }
 
 
-  metrics = concat([
-    { label = "In(container)", expression = "sum(rate(container_network_receive_bytes_total{namespace=\"${var.namespace}\", pod=~\"${var.container}-.*\"} [${var.period}m])) / 100000" },
-    { label = "Out(container)", expression = "sum(rate(container_network_transmit_bytes_total{namespace=\"${var.namespace}\", pod=~\"${var.container}-.*\"} [${var.period}m])) / 100000" },
-    ],
-    var.host != null ? [{ label = "In(ingress)", expression = "sum(rate(nginx_ingress_controller_bytes_sent_bucket{host=\"${var.host}\"}[${var.period}m]))" }] : [],
-  )
+  metrics = [
```

Comment:
Please, bring back the old queries, those were specified for some purpose. Also, make the labels in small letters(Received, Sent).